### PR TITLE
tariq commit

### PR DESCRIPTION
**Pull request by tariqulislam29**
my first pull request

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -2,14 +2,14 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.nubandroiddev.nubandroiddevelopers">
 
-    <uses-permission android:name="android.permission.INTERNET"/>
```
Comment (ShakibUddin) [inline app/src/main/AndroidManifest.xml:5]: same line below

File: app/src/main/res/layout/fragment_sign_up.xml [hunk 2]
```diff
@@ -63,6 +63,7 @@
                 app:layout_constraintStart_toStartOf="@+id/emailLayout"
                 app:layout_constraintTop_toBottomOf="@+id/emailLayout"
                 app:errorEnabled="true"
+                android:clickable="true"
```
Comment (ShakibUddin) [inline app/src/main/res/layout/fragment_sign_up.xml:66]: Why did u add clickable?

**Review by ShakibUddin — CHANGES_REQUESTED** [review]
why did u add clickable in the signup fragment layout?

**Comment by tariqulislam29** [comment]
hello

**Pull request merged**